common/linkerd-support/templates/namespace-annotation.yaml

```diff
@@ -57,7 +57,7 @@ spec:
       serviceAccountName: annotate-namespace-for-linkerd
       containers:
       - name: annotate
-        image: "{{ .Values.global.dockerHubMirror | required ".Values.global.dockerHubMirror not found" }}/bitnami/kubectl:latest"
+        image: "{{ .Values.global.registry | required ".Values.global.registry not found" }}/shared-app-images/alpine-kubectl:3.20-latest"
```

Comment:
Don't we have a latest-latest? the alpine update can't really break things here, can it?